qiskit/visualization/pulse_visualization.py

```diff
@@ -175,7 +173,7 @@ def pulse_drawer(data: Union[Waveform, Union[Schedule, Instruction]],
 
     if get_backend() in ['module://ipykernel.pylab.backend_inline',
                          'nbAgg']:
-        _matplotlib.plt.close(image)
+        plt.close(image)
```

Comment:
Is calling `close` on `qiskit.visualization.pulse.matplotlib` (which it was before) the same as calling it on `matplotlib` (which it is now)?

Reply:
It should be, [`matplotlib.pyplot`](https://matplotlib.org/3.3.3/api/_as_gen/matplotlib.pyplot.html#module-matplotlib.pyplot) is a global state instance.

That being said this comment reminded me that I need to check the pulse v2 drawer because I originally wrote this before that merged. Let me do a quick pass on that now and verify that matplotlib doesn't get imported at runtime from there.

Reply:
ok, just double checked and it looks like @nkanazawa1989 already made the matplotlib imports occur at runtime for the v2 drawer so there aren't any changes I need to make there.